.github/workflows/packages-build-wpk.yml

```diff
@@ -0,0 +1,98 @@
+name: Packages - Build WPK packages
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag name of the Docker image to be downloaded (Use developer to set branch name. Use auto to set src/VERSION content)'
```

Comment:
The way this input is set, allows values different from `developer` or `auto`, should we restrict it?

Reply:
I think yes, users could want add a tag name referring some custom names.